mundane-dsv/src/test/scala/com/ambiata/mundane/dsv/DelimitedValuesParserSpec.scala

```diff
@@ -0,0 +1,73 @@
+package com.ambiata.mundane.dsv
+
+import com.ambiata.disorder._
+import Arbitraries._
+import org.scalacheck._, Gen._, Arbitrary._
+import org.specs2._
+import org.specs2.matcher.DisjunctionMatchers
+
+class DelimitedValuesParserSpec extends Specification with ScalaCheck with DisjunctionMatchers { def is = s2"""
```

Comment:
Needs some benchmarks. Might be worth pulling in the scaffolding off of the benchmark branch @charleso is that possible?